lib/kwtool_web/templates/keyword/_keyword-result.html.eex

```diff
@@ -0,0 +1,46 @@
+<%= if length(@keyword_results) > 0 do %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th scope="col">Top Ads Count</th>
+        <th scope="col">Top Ads Urls</th>
+        <th scope="col">All Ads Count</th>
+        <th scope="col">Organic Result Count</th>
+        <th scope="col">Organic Result Urls</th>
+        <th scope="col">All Links Count</th>
+        <th scope="col">Inserted At</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for keyword_result <- @keyword_results do %>
+        <tr>
+          <td><%= keyword_result.top_ads_count %></td>
+          <td>
+            <%= if length(keyword_result.top_ads_urls) > 0 do %>
```

Comment:
Let's use `Enum.any?` here

Reply:
Updated in https://github.com/byhbt/kwtool/pull/44/commits/213a5dfa7b88ebb6f5e12f03c5b2ea849a9b56ca

Reply:
Updated in https://github.com/byhbt/kwtool/commit/026f5369772dd70e47d30f40105a139adb9a1ad7